datacatalog/pkg/manager/impl/artifact_manager_test.go

```diff
@@ -639,16 +640,23 @@ func TestUpdateArtifact(t *testing.T) {
 					artifactKey.DatasetVersion == expectedArtifact.Dataset.Version
 			})).Return(mockArtifactModel, nil)
 
+		metaData := &datacatalog.Metadata{
+			KeyMap: map[string]string{"key2": "value2"},
+		}
+		serializedMetadata, err := transformers.SerializedMetadata(metaData)
```

Comment:
looks like there is a lint err here because `err` is not used. maybe add `assert.Nil(...)` on the `err` here.